src/test/scala/nl/knaw/dans/easy/bagindex/components/DatabaseSpec.scala

```diff
@@ -35,9 +35,10 @@ class DatabaseSpec extends TestSupportFixture with BagIndexDatabaseFixture with
       DateTime.now()
     )
     val dois = List("10.5072/dans-x6f-kf6x", "10.5072/dans-x6f-kf66", "10.5072/dans-y7g-lg77")
+    val urns = List("urn:nbn:nl:ui:13-00-1haq", "urn:nbn:nl:ui:13-00-2haq", "urn:nbn:nl:ui:13-00-3haq")
 
-    (bagIds, times, dois).zipped.toList
-      .map { case (bagId, time, doi) => database.addBagInfo(bagId, baseId, time, doi) }
+    (bagIds zip times zip dois zip urns)
```

Comment:
what does this mean? It is repeated later also

Reply:
The method `zipped` works only up to three elements.  Now we have four, that is why I have to use zip...zip...zip